acceptance/setup/common/022_Remove_LD_PRELOAD.rb

```diff
@@ -0,0 +1,11 @@
+platforms = hosts.map{|val| val[:platform]}
+skip_test "No Cisco XR hosts present" unless platforms.any? { |val| /^cisco_ios_xr-/ =~ val }
+test_name 'Cisco XR Switch Pre-suite' do
+  switchs = select_hosts({:platform => ['cisco_ios_xr-6-x86_64']})
+
+  step 'remove LD_PRELOAD setting from switch' do
+    switchs.each do |switch|
+      on(switch, "echo 'unset LD_PRELOAD' >> /etc/profile")
```

Comment:
I think the "long term" (not so long anymore) fix is https://tickets.puppetlabs.com/browse/PA-437 - making sure that we have control over our loader environment whenever our programs/services are run.